docs/features/experiment.md

```diff
@@ -194,3 +194,54 @@ necessary metrics queries, using the `{{templates.baseline.podTemplateHash}}` an
     Rollout. This is despite the fact that the PodSpec are the same. This is intentional behavior,
     in order to allow the metrics of the Experiment's pods to be delineated and queried separately
     from the metrics of the Rollout pods.
+
+
+
+## Weighted Experiment Step with Traffic Routing
+!!! important
+    Available since v1.1
+
+A Rollout using the Canary strategy along with Traffic Routing can 
+split traffic to an experiment stack in a fine-grained manner. When
+Traffic Routing is enabled, the Rollout Experiment step allows
+traffic to be shifted to experiment pods.
+
+!!! note
+    This feature is currently available only for the SMI and ALB Traffic Routers.
+
+```yaml
```

Comment:
do you have examples for SMI and ALB added separately so that users can copy and run as is without changing any thing? we should at least add in getting started guide